addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -269,15 +269,15 @@
                                 <div class="row">
                                     <div class="col-lg my-2">
                                         <label>Name</label>
-                                        <input class="form-control" type="text" t-attf-name="#{counter}-name" required="This field is required"/>
+                                        <input class="form-control" type="text" t-attf-name="#{counter}-name" required="This field is required" t-att-value="default_first_attendee.get('name', '') if counter == 1 else ''"/>
```

Comment:
Just for readability, you can go on next line to avoid having to scroll too much :)